WORKSPACE

```diff
@@ -20,7 +20,10 @@ nixpkgs_git_repository(
   # lens-labels_0_2_0_0 to be available in nixpkgs. This means we need to
   # use a version of nixpkgs that is newer than 18.03.
 
-  # Keep this value in sync with `nixpkgs.nix`
+  # You need to be in an environment that is on the same commit
+    # as the one below. Use `nix-shell shell.nix`.
```

Comment:
Nitpicking, but could we align this comment so the indentation match the indentation level of the previous line?